packages/emoji-mart/src/components/Picker/Picker.tsx

```diff
@@ -1003,6 +1005,23 @@ export default class Picker extends Component {
     )
   }
 
+  getSelectedSkinToneName(skinTone) {
+    switch (skinTone) {
+      case '1':
+        return 'Light Skin Tone'
```

Comment:
This is not used, and it is correctly referenced above: https://github.com/slidoapp/emoji-mart/pull/4/files#diff-31f013abab5469ec1b60e7d41585ab05d6a53713ade4efb5e98d6aaf8171ab33R978, so this can be removed.